### PR TITLE
WEBUI-863: select2 in Spreadsheet Editor

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
https://jira.nuxeo.com/browse/WEBUI-863

### CODE REVIEW

File: addons/nuxeo-spreadsheet/app/lib/select2-editor.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ Select2Editor.prototype.open = function() {
 
   this.$textarea.css({
     height: $(this.TD).height(),
-    'min-width': $(this.TD).outerWidth(),
+    width: $(this.TD).width(),
```
Comment (richardsd) [inline addons/nuxeo-spreadsheet/app/lib/select2-editor.js:119]: I'm just wondering if replacing the `min-width` with `width` will not reveal any other problem 🤔

**Review by richardsd — DISMISSED** [review]

File: addons/nuxeo-spreadsheet/app/lib/select2-editor.js (hunk 1 above)
Comment (rakeshkumar1019) [reply addons/nuxeo-spreadsheet/app/lib/select2-editor.js:119]: Although I tested locally, Iam  keeping now both "width" and "min-width" just to be safe.

**Review by poonamyadav252 — COMMENTED** [review]
Mark as 'Closed' your open conversation if the issue is resolved and try to pass `function checks`.

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks good.

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by Nishant0928 — APPROVED** [review]

**Review by Pranit-Sotre — APPROVED** [review]

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good.

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by poonamyadav252 — APPROVED** [review]

File: addons/nuxeo-spreadsheet/app/lib/select2-editor.js [hunk 2]
```diff
@@ -116,7 +116,8 @@ Select2Editor.prototype.open = function() {
...
+    'min-width': $(this.TD).width(),
```
Comment (semisse) [inline addons/nuxeo-spreadsheet/app/lib/select2-editor.js:120]: why do we need both `min-width` and `width`?

Comment (rakeshkumar1019) [reply addons/nuxeo-spreadsheet/app/lib/select2-editor.js:120]: @semisse Thank you for your comment . 
Just to make sure that replacing 'min-width' with 'width' does not reveal any other issues.
@richardsd was thinking the same thing. Please see here https://github.com/nuxeo/nuxeo-web-ui/pull/1601#discussion_r979717527

**Review by rajatgoel-gl — APPROVED** [review]

File: addons/nuxeo-spreadsheet/app/lib/select2-editor.js (hunk 2 above)
Comment (semisse) [reply addons/nuxeo-spreadsheet/app/lib/select2-editor.js:120]: It's a little bit strange because one will override the other.

**Review by semisse — APPROVED** [review]

**Pull request merged**